media/doc/Chinese translation notes.txt

```diff
@@ -0,0 +1,56 @@
+The following material is addressed (in Chinese) to Chinese translators.
+
+------------------------------------------------------------------------
+（简体中文版本）
+欢迎翻译 ReactOS！在开始翻译之前，希望您可以注意以下几点以便更好地进行翻译。
+
+1）请注意您正在翻译的中文“变体”。当前翻译中的简体中文变体类別只有中国简体中文 (zh-CN)。
+2）Windows 的记事本会将 UTF-8 格式变成 UTF-8 BOM 格式。因此，请不要用记事本编辑 ReactOS 的文件，特别是翻译文件。您可以使用 Notepad++ 之类的第三方文本编辑器进行编辑。
+3）如果您是在 macOS 下进行翻译工作的，因为 macOS 的特性，子目录下面可能会产生 .DS_Store，所以需要在 git 提交前在代码根目录中执行 find . -name ".DS_Store" -depth -exec rm {} \; 以删除 .DS_Store。
+4）请不要将 USERENV 和 SHELL32 里面所提到的路径擅自翻译成中文，里面一些路径是可以翻译的，但是也有很多的路径是不可以进行翻译的，翻译的话会产生程序兼容性问题。曾经有段时间 ReactOS 里的路径都变成了中文，笔者在运行某个程序的时候就出现了问题。
+5）为了翻译的准确性，您可以适当参考 Windows 中的对应条目的翻译，但是请不要照搬 Windows 的翻译，或者大段抄袭。这样可能会产生版权争议导致法律问题。
+6）除特定字词（如 ReactOS）需保留原文外，字符串不可以中英夹杂。
+7）来自 Wine 的翻译文件有特殊的翻译规则:
+7.1: 它们大多使用旧的 "(filename_)Zh.rc" 命名格式而非新的 "xx-YY.rc" 格式。如非必要，请不要随意更改命名格式。
+7.2: 不论您是在翻译何种变体，当您翻译来自 Wine 的文件的时候，均需要把所有中文变体的翻译放在同一个翻译文件。除非该文件已使用新的 "xx-YY.rc" 命名格式。
+7.3: 如果您翻译的 Wine 文件是首次翻译，则只需要同时在上一个目录的 "(filename_).rc" 或 "rsrc.rc" 中加入 "zh-CN" 的 #ifdef 数据即可。除非该文件已使用新的 "xx-YY.rc" 命名格式。
+7.4: 由于这些文件大多是繁体与简体并存。因此请在新增翻译文件前先行确定是否已有相关文件。
+7.5: 翻译时，您需要确保翻译与 Wine.hq 的翻译文件同步。
+
+香港繁体中文 (zh-HK) 翻译注意事项：
+H1）所有翻译必须使用繁体字，不接受使用简体字。校对员会拒绝任何含有简体字的港式中文翻译。
+**至于其他规则，请浏览正体中文版本。**
+
+其他翻译细则可以到 ReactOS Wiki 查看。
+
+祝您翻译愉快！
+
+
+（正體中文版本）
+歡迎翻譯 ReactOS！在開始翻譯前，希望您可以注意以下幾點以便更好地進行翻譯︰
+
+1）請注意您正在翻譯的中文「變體」。目前翻譯中的繁體中文變體類別有︰臺灣正體中文 (zh-TW) 和香港繁體中文 (zh-HK)。
+2）Windows 內置的記事本會將 UTF-8 格式變為 UTF-8 BOM 格式。因此，請勿使用記事本編輯 ReactOS 的檔案，尤其是翻譯檔案。您可使用 Notepad++ 之類的第三方文字編輯器進行編輯。
+3）如果您是在 macOS 下進行翻譯，因為 macOS 的特性，子目錄下面可能會產生 .DS_Store，所以需要在 git 提交前在代碼根目錄中執行 find . -name ".DS_Store" -depth - exec rm {} \; 以刪除 .DS_Store。
+4）請勿將 USERENV 和 SHELL32 裡面所提到的路徑擅自翻譯為中文。很多的路徑都是不可以翻譯的（即便有一些路徑還是可以翻譯的），否則會產生程序相容性問題。曾經有段時間 ReactOS 裡的路徑都變成了中文，部分使用者在執行特定程序的時候就出現了錯誤。
+5）為了翻譯的準確性，您可以適當參考 Windows 中的對應條目的翻譯，但請不要照搬 Windows 的翻譯或大量抄襲。這樣可能會產生著作權爭議，進而導致法律問題。
+6）除特定字詞（如 ReactOS）需保留原文外，字串不可以中英夾雜。
+7）來自 Wine 的翻譯檔案有特殊的翻譯規則:
+7.1: 它們大多使用舊的 "(filename_)Zh.rc" 命名格式而非新的 "xx-YY.rc" 格式。如非必要，請勿隨意更改命名格式。
+7.2: 除非該檔案已使用新的 "xx-YY.rc" 命名格式，否則，您均需要把所有中文變體的翻譯放在同一個翻譯檔案。另請不要以 "(filename_)Tw.rc" 新增正體中文翻譯檔案。因為這名稱與契維語（Twi）的語言代碼有衝突。
+7.3: 如果您翻譯的 Wine 檔案是首次翻譯，則只需要同時在上一個目錄的 "(filename_).rc" 或 "rsrc.rc" 中加入 "zh-CN" 的 #ifdef 資料即可。除非該檔案已使用新的 "xx-YY.rc" 命名格式。
+7.4: 由於這些檔案大多是繁體與簡體並存。因此請在新增翻譯檔案前先行確定是否已有相關檔案。
+7.5: 翻譯時，您需要確保翻譯與 Wine.hq 的翻譯檔案同步。
+
+香港繁體中文 (zh-HK) 翻譯注意事項：
+H1）所有翻譯必須使用繁體（正體）字，不可使用簡體字。校對員會拒絕任何含有簡體字的港式中文翻譯。
+H2）請使用港式中文書面語翻譯。只有在特殊情況下才可使用粵文 (Written Cantonese)，並需最大程度保留其可讀性。
+H3）香港繁體中文所使用的字形以《香港小學學習字詞表》為標準。
+H4）所有習慣稱呼（例如: 取消、網絡）需要以香港和／或澳門為標準。如兩地習慣稱呼有異，則以前者為準。
+H5）由於 Wine 不支援香港繁體中文，大部份與 Wine 相關的翻譯須保留英文原文（即不翻譯）。如該檔案有特殊原因而需要翻譯，則需使用正體中文版本的翻譯。翻譯時請跟隨上述規則。
+
+其他翻譯細則可以到 ReactOS Wiki 查看。
+
+祝您翻譯愉快！
+
+（最近更新日期：2021 年 11 月 20 日）
```

Comment:
If you want to rename this file, better do it in separate commit, for clearer history.